package.json

```diff
@@ -157,7 +157,8 @@
     "@storybook/addon-viewport": "^5.3.18",
     "@storybook/react": "^5.3.18",
     "@testing-library/jest-dom": "^4.1.2",
-    "@testing-library/react": "^9.4.1",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/react-hooks": "^3.3.0",
```

Comment:
`@testing-library/react` was updated to fix a problem with `getByRole` (and perhaps other queries) not supporting a text match on the older version. It's a minor bump from a package I think we can trust to adhere to proper semantic versioning so it should be safe but otherwise I wouldn't have updated it here other than to fix what seemed like a bug.

`@testing-library/react-hooks` is just an addition which is necessary for testing the added hook here. Since it's only used for that it shouldn't affect any other tests.